style.css

```diff
@@ -96,3 +96,138 @@ a {
 .social-media li img {
   height: 20px;
 }
+
+.cards-container {
+  background-color: #e5e5e5;
+  padding: 11rem 1.5rem;
+  display: grid;
+  grid-template-rows: repeat(3, 1fr);
+  gap: 12rem;
+}
+
+.card {
+  background-color: white;
+  padding: 2.5rem 1.5rem;
+  border-radius: 25px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
+
+.card-image {
+  width: 100%;
+}
+
+.card h2 {
+  color: var(--black);
+  font-size: 3.2rem;
+  margin: 1rem 0;
+}
+
+.tags {
+  display: flex;
+  align-items: center;
+  gap: 0.7rem;
+}
+
+.bullet {
+  width: 6px;
+}
+
+.tags p {
+  color: var(--black);
+  font-weight: 700;
+}
+
+.tags span {
+  color: #7a869a;
+}
+
+.description {
+  color: #344563;
+  font-size: 1.6rem;
+}
+
+.technologies {
+  list-style: none;
+  padding-left: 0;
+  display: flex;
+  gap: 1rem;
+}
+
+.technologies li {
+  background-color: #ebebff;
+  padding: 0.8rem;
+  color: #6070ff;
+  border-radius: 8px;
+  font-size: 1.2rem;
+}
+
+.card button {
+  color: #396df2;
+  border: 2px solid #396df2;
+  background-color: white;
+  border-radius: 8px;
+  padding: 1rem;
+  flex: none;
```

Comment:
- [ ] I noticed the borders on your buttons are thicker than the ones in Figma template. Kindly fix this, please. You are expected to stick to design as much as possible.
### Your design 
![1](https://user-images.githubusercontent.com/94551484/212396281-40e1b375-4e20-4791-b720-4d782c5eb17b.png)
### Figma template
![f1](https://user-images.githubusercontent.com/94551484/212396288-b44921b5-33b7-4241-8140-ea4df41cebd6.png)

- [ ] Kindly add a hover and pressed state to all your buttons to match the figma template. You can use the `:hover` and `:active` pseudo-classes.
![btn1](https://user-images.githubusercontent.com/94551484/212396453-f4d44148-5182-4545-bd9c-1d1abc038423.png)